### PR TITLE
docs: PR/DN Return Tracking

### PR DESCRIPTION
**Pull request by marination**
Documentation for https://github.com/frappe/erpnext/pull/22859

### CODE REVIEW

File: erpnext_documentation/www/docs/user/manual/en/stock/delivery-note.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ These are the statuses a Delivery Note can be in:
 * **Draft**: A draft is saved but yet to be submitted to the system.
 * **To Bill**: Yet to be billed using a [Sales Invoice](/docs/user/manual/en/accounts/sales-invoice).
 * **Completed**: Submitted and sent all the Items.
+* **Return Issued**: All the Items have been returned.
 * **Canceled**: Canceled the Delivery Note.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/stock/delivery-note.md:50]: **Cancelled**: Cancelled the Delivery Note.

File: erpnext_documentation/www/docs/user/manual/en/stock/purchase-receipt.md [hunk 2]
```diff
@@ -47,6 +47,7 @@ These are the statuses a Purchase Receipt can be in:
 * **Draft**: A draft is saved but yet to be submitted to the system.
 * **To Bill**: Yet to be billed using a [Purchase Invoice](/docs/user/manual/en/accounts/purchase-invoice).
 * **Completed**: Submitted and received all the Items.
+* **Return Issued**: All the Items have been returned.
 * **Canceled**: Canceled the Purchase Receipt.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/stock/purchase-receipt.md:51]: **Cancelled**: Cancelled the Purchase Receipt.

File: erpnext_documentation/www/docs/user/manual/en/stock/purchase-return.md [hunk 3]
```diff
@@ -37,6 +37,17 @@ Before creating and using a Purchase Return, it is advised that you create the f
...
-### 3. Related Topics
+## 3. Impact on Stock Return via Purchase Receipt
+On Creating a Purchase Return against a Purchase Receipt:
+
+* The **Returned Quantity** in the original Purchase Receipt along with any Purchase Order linked to it, is updated.
+
+* The original Purchase Receipt's status is changed to **Return Issued** if 100% returned.
+  ![Return Issued](/docs/assets/img/stock/purchase-return-issue.png)
+
+* The Returned Amount % is visible via the List View as well
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/stock/purchase-return.md:48]:
```suggestion
* The Returned Amount % is visible via the List View as well:
```

File: erpnext_documentation/www/docs/user/manual/en/stock/sales-return.md [hunk 4]
```diff
@@ -39,6 +39,17 @@ Before creating and using a Sales Return, it is advised that you create the foll
...
-## 3. Related Topics
+## 3. Impact on Stock Return via Delivery Note
+On Creating a Sales Return against a Delivery Note:
+
+* The **Returned Quantity** in the original Delivery Note along with any Sales Order linked to it, is updated.
+
+* The original Delivery Note's status is changed to **Return Issued** if 100% returned.
+  ![Return Issued](/docs/assets/img/stock/sales-return-issue.png)
+
+* The Returned Amount % is visible via the List View as well
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/stock/sales-return.md:50]:
```suggestion
* The Returned Amount % is visible via the List View as well:
```

File: erpnext_documentation/www/docs/user/manual/en/stock/sales-return.md [hunk 5]
```diff
@@ -39,6 +39,17 @@ Before creating and using a Sales Return, it is advised that you create the foll
 
 If Perpetual Inventory is enabled, the system will also post accounting entry against warehouse account to sync warehouse account balance with stock balance as per Stock Ledger.
 
-## 3. Related Topics
+## 3. Impact on Stock Return via Delivery Note
+On Creating a Sales Return against a Delivery Note:
+
+* The **Returned Quantity** in the original Delivery Note along with any Sales Order linked to it, is updated.
+
+* The original Delivery Note's status is changed to **Return Issued** if 100% returned.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/stock/sales-return.md:47]:
```suggestion
* The original Delivery Note's status is changed to **Return Issued** if 100% returned:
```

File: erpnext_documentation/www/docs/user/manual/en/stock/purchase-return.md [hunk 6]
```diff
@@ -37,6 +37,17 @@ Before creating and using a Purchase Return, it is advised that you create the f
 
 If Perpetual Inventory enabled, the system will also post accounting entry against warehouse account to sync warehouse account balance with stock balance as per Stock Ledger.
 
-### 3. Related Topics
+## 3. Impact on Stock Return via Purchase Receipt
+On Creating a Purchase Return against a Purchase Receipt:
+
+* The **Returned Quantity** in the original Purchase Receipt along with any Purchase Order linked to it, is updated.
+
+* The original Purchase Receipt's status is changed to **Return Issued** if 100% returned.
```
Comment (prasadarr) [inline erpnext_documentation/www/docs/user/manual/en/stock/purchase-return.md:45]:
```suggestion
* The original Purchase Receipt's status is changed to **Return Issued** if 100% returned:
```

**Review by prasadarr — CHANGES_REQUESTED** [review]

File: erpnext_documentation/www/docs/user/manual/en/stock/purchase-return.md (hunk 3 above)
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/stock/purchase-return.md:48]: add a full stop at the end

File: erpnext_documentation/www/docs/user/manual/en/stock/purchase-return.md [hunk 7]
```diff
@@ -37,6 +37,17 @@ Before creating and using a Purchase Return, it is advised that you create the f
...
+  ![Return Issued](/docs/assets/img/stock/purchase-return-issue.png)
+
+* The Returned Amount % is visible via the List View as well
+  ![Purchase Receipt List View](/docs/assets/img/stock/purchase-receipt-list.png)
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/stock/purchase-return.md:49]: reduce the left-hand side border (for all images)

File: erpnext_documentation/www/docs/user/manual/en/stock/sales-return.md [hunk 8]
```diff
@@ -3,14 +3,14 @@
 
 **A sold Item being returned is known as a Sales Return.**
 
-Goods sold being returned happens commonly in business. They could be returned by the customer due to quality issues, non-delivery on the agreed date, or any other reason. 
+Goods sold being returned happens commonly in business. They could be returned by the customer due to quality issues, non-delivery on the agreed date, or any other reason.
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/stock/sales-return.md:6]: Businesses often return goods that are already sold.

File: erpnext_documentation/www/docs/user/manual/en/stock/sales-return.md (hunk 4 above)
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/stock/sales-return.md:50]: add a full stop at the end

File: erpnext_documentation/www/docs/user/manual/en/stock/sales-return.md [hunk 9]
```diff
@@ -39,6 +39,17 @@ Before creating and using a Sales Return, it is advised that you create the foll
...
+## 3. Impact on Stock Return via Delivery Note
+On Creating a Sales Return against a Delivery Note:
+
+* The **Returned Quantity** in the original Delivery Note along with any Sales Order linked to it, is updated.
+
+* The original Delivery Note's status is changed to **Return Issued** if 100% returned.
+  ![Return Issued](/docs/assets/img/stock/sales-return-issue.png)
+
+* The Returned Amount % is visible via the List View as well
+  ![Purchase Receipt List View](/docs/assets/img/stock/delivery-note-list.png)
```
Comment (michellealva) [inline erpnext_documentation/www/docs/user/manual/en/stock/sales-return.md:51]: shouldn't this be.. Delivery Note List View ?

**Review by michellealva — CHANGES_REQUESTED** [review]

File: erpnext_documentation/www/docs/user/manual/en/stock/purchase-return.md (hunk 7 above)
Comment (marination) [reply erpnext_documentation/www/docs/user/manual/en/stock/purchase-return.md:49]: looks okay to me @prasadarr should i ?

**Review by prasadarr — APPROVED** [review]

**Pull request merged**